deployment/src/main/java/io/quarkiverse/authzed/client/deployment/DevServicesAuthzedConfig.java

```diff
@@ -118,5 +118,13 @@ public class DevServicesAuthzedConfig {
      */
     @ConfigItem
     public MetricsConfig metrics;
+    
+     /**
+     * operationType configuration
+     */
+    @ConfigItem(defaultValue = "CREATE")
```

Comment:
`CREATE` is not a valid value for the `Operation` enum (which is how this is used).
It should be `OPERATION_CREATE` instead.

Reply:
```suggestion
    @ConfigItem(defaultValue = "OPERATION_CREATE")
```